apps/src/codebridge/FileBrowser/InnerFileBrowser.tsx

```diff
@@ -52,12 +52,16 @@ const InnerFileBrowser = React.memo(
                 key={f.id + f.open}
                 Component="div"
                 className={classNames(moduleStyles.droppableArea, {
-                  [moduleStyles.acceptingDrop]:
-                    f.id === dropData?.id && dragData?.parentId !== f.id,
+                  [moduleStyles.acceptingDrop]: f.id === dropData?.id,
                 })}
               >
                 <MaybeDraggable
                   data={{id: f.id, type: DragType.FOLDER, parentId: f.parentId}}
+                  className={
+                    f.id === dragData?.id && dragData?.type === DragType.FOLDER
```

Comment:
The PR description says we have styling for draggable folders and files- does this just handle the folder case?

Reply:
This is the folder, you can see the file case change on line 96.